CONTRIBUTING.md

```diff
@@ -20,9 +20,9 @@ To check your the project's dependencies run:
 $ scripts/check_setup.sh
 ```
 
-## Building the Unity Buy SDK
+## Building the Shopify SDK for Unity
 
-Before you can start developing on the SDK you'll need to run the code generator. Under the hood, the Unity Buy SDK works on top of GraphQL. In order to query Shopify, we generate a GraphQL client based on the [Storefront API](https://help.shopify.com/api-storefront-api) introspection schema.
+Before you can start developing on the SDK you'll need to run the code generator. Under the hood, the Shopify SDK for Unity works on top of GraphQL. In order to query Shopify, we generate a GraphQL client based on the [Storefront API](https://help.shopify.com/api-storefront-api) introspection schema.
```

Comment:
I know this is outside of the scope of this PR but I wonder if we should say something like.

> If you do not want to download a Unity Package you have the option of building source that you can import into your project.